src/Analysis/Ast/Impl/Analyzer/Symbols/ClassEvaluator.cs

```diff
@@ -118,6 +111,51 @@ internal sealed class ClassEvaluator : MemberEvaluator {
             UpdateClassMembers();
         }
 
+        private void ProcessBases(List<IPythonType> bases, Scope outerScope) {
+            foreach (var a in _classDef.Bases.Where(a => string.IsNullOrEmpty(a.Name))) {
+                var expr = a.Expression;
+
+                switch (expr) {
+                    // class declared in the current module
+                    case NameExpression nameExpression:
+                        var name = Eval.GetInScope(nameExpression.Name, outerScope);
+                        switch (name) {
+                            case PythonClassType classType:
+                                bases.Add(classType);
+                                break;
+                            case IPythonConstant constant:
+                                ReportInvalidBase(constant.Value);
+                                break;
+                            default:
+                                TryAddBase(bases, a);
+                                break;
+                        }
+                        break;
+                    default:
+                        TryAddBase(bases, a);
+                        break;
+                }
+            }
+        }
+
+        private IPythonType TryAddBase(List<IPythonType> bases, Arg arg) {
```

Comment:
The return value for the function isn't used anywhere; did you mean to?

Reply:
good catch that's my bad